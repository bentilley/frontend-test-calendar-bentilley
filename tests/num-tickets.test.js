import React from 'react';
import renderer from 'react-test-renderer';

import NumTickets from 'components/num-tickets/num-tickets';

describe('NumTickets component', () => {
  it('renders correctly', () => {
    const component = renderer.create(<NumTickets />);
    const instance = component.getInstance();
    const result = instance.render();
    expect(result).toMatchSnapshot();
  });
});
