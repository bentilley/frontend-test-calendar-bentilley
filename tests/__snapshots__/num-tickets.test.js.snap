// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NumTickets component renders correctly 1`] = `
<div
  className="num-tickets"
>
  <p>
    1. Choose number of tickets
  </p>
  <div
    className="num-tickets-container"
  >
    <Choice
      number={1}
      selected={false}
      updateSelected={[Function]}
    />
    <Choice
      number={2}
      selected={false}
      updateSelected={[Function]}
    />
    <Choice
      number={3}
      selected={false}
      updateSelected={[Function]}
    />
    <Choice
      number={4}
      selected={false}
      updateSelected={[Function]}
    />
    <Choice
      number={5}
      selected={false}
      updateSelected={[Function]}
    />
    <LastChoice
      currentSelection={undefined}
      numAdditionalOptions={4}
      number={6}
      selected={false}
      updateSelected={[Function]}
    />
  </div>
</div>
`;
